core/src/main/java/com/scalar/db/api/Get.java

```diff
@@ -133,6 +133,11 @@ public Get withProjections(Collection<String> projections) {
     return (Get) super.withProjections(projections);
   }
 
+  @Override
+  Get withConjunctions(Collection<Conjunction> conjunctions) {
```

Comment:
Should we add `conjunctions` to the `toString()` method?

Reply:
Good catch, thank you! Fixed in e9840a9.